service/src/integrationTest/resources/fixtures/case.json

```diff
@@ -1,5 +1,16 @@
 {
   "case_details": {
+    "id": "2313",
+    "case_data": {
+      "caseLocalAuthority": "example",
+      "orders": {
+        "orderType": [
+          "EMERGENCY_PROTECTION_ORDER"
+        ]
+      }
+    }
+  },
+  "case_details_before": {
```

Comment:
Please remove `case_details_before`, we do not need it anymore. It just makes fixture unnecessarily more complex.